Hippo.Web/ClientApp/src/Shared/DebouncedInput.tsx

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+
+export const DebouncedInput = ({
```

Comment:
this is interesting -- I can't tell from looking at it if there will be any issues w/ onChange not being useCallback() or other weird rendering side effects.  I've always just used the `useDebounce()` hook for this purpose (https://github.com/uidotdev/usehooks/blob/main/index.js#L239) which is simple to reason over.  If it works though I think it's a pretty nice solution.